developer/src/kmc-keyboard-info/test/test-keyboard-info-compiler-messages.ts

```diff
@@ -234,9 +186,9 @@ describe('KeyboardInfoCompilerMessages', function () {
   // ERROR_LicenseIsNotValid
 
   it('should generate ERROR_LicenseIsNotValid error if license file is invalid', async function() {
-    const jsFilename = makePathToFixture('khmer_angkor', 'build', 'khmer_angkor.js');
-    const kpsFilename = makePathToFixture('khmer_angkor', 'source', 'khmer_angkor.kps');
-    const kmpFilename = makePathToFixture('khmer_angkor', 'build', 'khmer_angkor.kmp');
+    const jsFilename = KHMER_ANGKOR_JS;
+    const kpsFilename = KHMER_ANGKOR_KPS;
+    const kmpFilename = KHMER_ANGKOR_KMP;
 
     const sources = {
```

Comment:
You could do this as:

```js
const sources = {
  ...KHMER_ANGKOR_SOURCES,
  sourcePath: 'release/k/invalid-license',
};
```

here, and above, and that way eliminate a bunch of repeated constants. :grin:

Reply:
I found only a couple of places I could do this; can you spot any more?

Reply:
didn't spot any more!